easybuild/easyconfigs/i/intel/intel-2016.00.eb

```diff
@@ -0,0 +1,22 @@
+# This is an easyconfig file for EasyBuild, see http://hpcugent.github.io/easybuild
+easyblock = "Toolchain"
+
+name = 'intel'
+version = '2016.00'
```

Comment:
Hmm, I see your point. The `.00` just seems weird to me, what does that mean? initial release?

I don't feel really strongly about it, just as long as we can motivate whatever we come up with